R/ospsuite-env.R

```diff
@@ -20,7 +20,7 @@ ospsuiteEnv$formatNumericsDigits <- 5
 ospsuiteEnv$formatNumericsSmall <- 2
 
 # Number of cores to use for simualtions and sensitivity. Default to number of cores on the machine - 1
-ospsuiteEnv$numberOfCores <- parallel::detectCores() - 1
+ospsuiteEnv$numberOfCores <- function(){parallel::detectCores() - 1}

```

Comment:
I think the value was assigned upon building the package. It has to be a function that is called every time the value is asked for.